src/runtime/components/overlays/Notifications.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :class="wrapperClass" v-bind="attrs">
+  <div :class="wrapperClass" aria-live="assertive" v-bind="attrs">
```

Comment:
Shouldn't it be `role="region"` here? I'm taking https://ui.shadcn.com/docs/components/toast as example.

Reply:
I'll look into this